src/docs/common/snippets/common-openapi-dependencies.adoc

```diff
@@ -0,0 +1,11 @@
+== Micronaut OpenAPI
+
+To use https://micronaut-projects.github.io/micronaut-openapi/latest/guide/[Micronaut OpenAPI], add the `micronaut-openapi` annotation processor and the
+https://micronaut-projects.github.io/micronaut-openapi/latest/guide/#micronautOpenApiAnnotations[Micronaut OpenAPI annotations] as compile only dependency:
+
+:dependencies:
+dependency:micronaut-openapi[groupId=io.micronaut.openapi,scope=annotationProcessor]
+dependency:micronaut-openapi-annotations[groupId=io.micronaut.openapi]
```

Comment:
This isn't what starter generates...  Starter generates:

```
    annotationProcessor("io.micronaut.openapi:micronaut-openapi")
    implementation("io.swagger.core.v3:swagger-annotations")
```

See here https://github.com/micronaut-projects/micronaut-starter/blob/4.2.x/starter-core/src/main/java/io/micronaut/starter/feature/other/OpenApi.java#L68-L69

Reply:
Yes, I realised that yesterday and I created an issue: 

https://github.com/micronaut-projects/micronaut-starter/issues/2203